src/lib/locale/locales.js

```diff
@@ -142,6 +142,14 @@ export function defineLocale (name, config) {
 
 export function updateLocale(name, config) {
     if (config != null) {
+        if (locales[name] === undefined) {
```

Comment:
I'm not sure this is the right approach here. `loadLocale` should be used instead, because the `locales` object might not be filled yet.